app/src/full/kotlin/io/homeassistant/companion/android/settings/wear/SettingsWearViewModel.kt

```diff
@@ -51,12 +54,18 @@ class SettingsWearViewModel @Inject constructor(private val serverManager: Serve
 
     companion object {
         private const val CAPABILITY_WEAR_SENDS_CONFIG = "sends_config"
+        private const val CAPABILITY_WEAR_APP = "verify_wear_app"
     }
 
     private val _hasData = MutableStateFlow(false)
     val hasData = _hasData.asStateFlow()
     private val _isAuthenticated = MutableStateFlow(false)
     val isAuthenticated = _isAuthenticated.asStateFlow()
+    private val _wearNodesWithApp: MutableStateFlow<Set<Node>?> = MutableStateFlow(setOf<Node>())
+    val wearNodesWithApp = _wearNodesWithApp.asStateFlow()
+    private val _allConnectedNodes: MutableStateFlow<List<Node>?> = MutableStateFlow(listOf<Node>())
```

Comment:
I think you can remove the nullable from the types, by only calling the VM is the client is not null.
```suggestion
    private val _wearNodesWithApp: MutableStateFlow<Set<Node>> = MutableStateFlow(setOf<Node>())
    val wearNodesWithApp = _wearNodesWithApp.asStateFlow()
    private val _allConnectedNodes: MutableStateFlow<List<Node>> = MutableStateFlow(listOf<Node>())
```